packages/edit-site/src/components/dataviews/style.scss

```diff
@@ -34,9 +34,15 @@
 	}
 	th {
 		text-align: left;
-		font-weight: normal;
-		padding: 0 $grid-unit-20 $grid-unit-20;
-		color: $gray-700;
+		/*
+		 * This makes the Actions column header to use the same styles as the other ones.
+		 * The other column headers use the .components-button styles.
+		 */
+		&[data-field-id="actions"] {
+			color: #1e1e1e;
```

Comment:
I pinpointed this value to [this](https://github.com/WordPress/gutenberg/blob/update/actions-style/packages/components/src/utils/theme-variables.scss#L14). How do we reference colors that are defined in the `components` package. Should I use this CSS instead?

```css
color: var(--wp-components-color-foreground, $gray-900);
```

Reply:
I saw we are using the CSS custom properties in other places but not the sass variable, so I went with that.